tests/input/input.js

```diff
@@ -53,16 +48,18 @@ function createInput(component) {
   const originalRef = component.ref;
   let { props } = component;
   let input;
-  component = React.cloneElement(component, {
-    ref: ref => {
-      input = ref;
+  const refCallback = node => {
+    input = node;
```

Comment:
Rename `ref` --> `node` to match react docs. It is the `node` which is the equivalent to `ref.current`.